pkg/services/featuremgmt/registry.go

```diff
@@ -533,5 +533,12 @@ var (
 			State:        FeatureStateBeta,
 			Owner:        grafanaBiSquad,
 		},
+		{
+			Name:         "alertingLokiRangeToInstant",
+			Description:  "Rewrites eligible loki range queries to instant queries",
+			State:        FeatureStateAlpha,
```

Comment:
This was called "stage" on main but is here its still called "state". It has not the value "experimental" available, so I used alpha.